accounts/base/src/main/java/pm/gnosis/heimdall/accounts/base/models/Signature.kt

```diff
@@ -0,0 +1,13 @@
+package pm.gnosis.heimdall.accounts.base.models
+
+import java.math.BigInteger
+
+
+data class Signature(val r: BigInteger, val s: BigInteger, val v: Byte) {
+    override fun toString(): String {
+        return r.toString(16).padStart(64, '0') +
+                s.toString(16).padStart(64, '0') +
+                v.toString(16).padStart(2, '0')
+
+    }
+}
```

Comment:
Should we verify if the parameters are 32 bytes? Maybe in the `init` method

Reply:
I was thinking about it but didn't deem it necessary, since that is really dependent on the implementation. The toString method is more for debugging purpose and not to use as valid encoding